src/toil_scripts/gatk_germline/hard_filter.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python2.7
```

Comment:
Why is this in its own .py file?  It seems like this and the other file either belong in toil-lib or should be in the germline source.